src/transformers/models/llama/modeling_llama.py

```diff
@@ -831,3 +824,105 @@ def _reorder_cache(past_key_values, beam_idx):
         for layer_past in past_key_values:
             reordered_past += (tuple(past_state.index_select(0, beam_idx) for past_state in layer_past),)
         return reordered_past
+
+
+class LlamaForSequenceClassification(LlamaPreTrainedModel):
+    _keys_to_ignore_on_load_missing = [r"lm_head.weight"]
```

Comment:
I'm not entirely sure which other keys should be ignored for the LLaMa implementation - some guidance here is appreciated!

Reply:
I think that it should be similar to the ones in `LlamaForCausalLM`